gips/data/sentinel2/sentinel2.py

```diff
@@ -800,14 +806,20 @@ def ref_toa_geoimage(self, sensor, data_spec):
         with utils.make_temp_dir() as tmpdir:
             upsampled_filenames = [os.path.join(tmpdir, os.path.basename(f) + '.tif')
                     for f in src_filenames]
+            #  TODO:  this should be stuffed into the 'env' of Popen, but is a
+            #         necessary hack for now.
+            os.putenv(
+                'GDAL_NUM_THREADS',
+                str(int(gippy.Options.NumCores()))
+            )
             for in_fn, out_fn in zip(src_filenames, upsampled_filenames):
-                cmd_str = 'gdalwarp -tr 10 10 {} {}'.format(in_fn, out_fn)
+                cmd_str = 'gdal_translate -tr 20 20 -oo NUM_THREADS=1 {} {}'.format(in_fn, out_fn)
```

Comment:
I'll add the skip and an issue to convert the sentinel2 driver to use gdal-VRT all over the place.